src/iuse_actor.cpp

```diff
@@ -690,6 +690,21 @@ static effect_data load_effect_data( const JsonObject &e )
                         get_body_part_token( e.get_string( "bp", "NUM_BP" ) ), e.get_bool( "permanent", false ) );
 }
 
+static DoT_data load_DoT_data( const JsonObject &e )
```

Comment:
rather than a static function, this should be a member function load() like how most deserialization is done.